src/applications/disability-benefits/526EZ/tests/config/ratedDisabilities.unit.spec.jsx

```diff
@@ -55,12 +55,10 @@ describe('Disability benefits 526EZ -- Rated disabilities selection', () => {
     );
 
     const labels = form.find('input[type="checkbox"] + label');
-    expect(labels.at(0).find('h4').text()).to.equal('PTSD');
-    expect(labels.at(0).find('.diagnostic-text').text()).to.contain('Post traumatic stress disorder');
+    expect(labels.at(0).find('h4').text()).to.contain('PTSD');
```

Comment:
NBD, but might want to test for the full string since we know what it is from the mapping.